### PR TITLE
chore(*): update ruby and sinatra versions

### PR DESCRIPTION
**Pull request by mboersma**
Also recreated the README.md example output to match the current Deis buildpack. Tested with Deis master.

### CODE REVIEW

File: web.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 set :port, ENV["PORT"] || 5000
 
 get '/' do
-  whom = ENV["POWERED_BY"] || "Deis!"
+  whom = ENV["POWERED_BY"] || "Deis"
```
Comment (sgoings) [inline web.rb]: It's not quite as excited anymore. :cry:

File: Gemfile [hunk 2]
```diff
@@ -1,4 +1,3 @@
 source 'http://rubygems.org'
-ruby '1.9.3'
+ruby '2.0.0'
 gem 'sinatra'
-gem 'rack'
```
Comment (sgoings) [inline Gemfile]: No more rack requirement?

**Comment by mboersma** [comment]
Yes i think `rack` comes through `sinatra` now:
```
       Installing rack 1.6.1
       Installing rack-protection 1.5.3
```

File: Gemfile (hunk 2 above)
Comment (mboersma) [inline Gemfile]: Yes i think `rack` comes through `sinatra` now:
```
       Installing rack 1.6.1
       Installing rack-protection 1.5.3
```

**Pull request merged**